contracts/market/template/MarketExtended.sol

```diff
@@ -132,13 +149,13 @@ contract MarketExtendedCore is AccessControlledAndUpgradeableModifiers, MarketSt
     );
 
     SinglePoolInitInfo memory poolInfo = initPool;
-    uint256 tierPriceMovementThresholdAbsolute = PoolType.FLOAT == initPool.poolType ? 0.1999e18 : (1e36 / poolInfo.leverage) - 1e14;
+    uint256 tierPriceMovementThresholdAbsolute = PoolType.FLOAT == initPool.poolType ? 0.19e18 : (1e36 / poolInfo.leverage) - 1e16;
```

Comment:
Adjusted the max price movement to be at a higher threshold - prudence - nothing 'wrong' with the old value.